plugin/gitsessions.vim

```diff
@@ -65,37 +65,45 @@ function! s:os_sep()
 endfunction
 
 function! s:is_abs_path(path)
-    return a:path[0] == s:os_sep()
+    if has('unix')
+        return a:path[0] == s:os_sep()
+    else
+        return a:path[1] == ':'
+    endif
 endfunction
 
 " LOGIC FUNCTIONS
 
 function! s:parent_dir(path)
     let l:sep = s:os_sep()
-    let l:front = s:is_abs_path(a:path) ? l:sep : ''
+    let l:front = s:is_abs_path(a:path) && has('unix') ? l:sep : ''
```

Comment:
Doesn't the `has('unix')` check make the `is_abs_path(a:path)` changes unnecessary since it'll never be true on Windows?

The result is all absolute paths are converted into relative paths on Windows, regardless if it was an absolute path to begin with or not.